### PR TITLE
Pull request for Analytics screen

### PR DESCRIPTION
**Pull request by nj-app**

### CODE REVIEW

File: App1.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
+	<true/>
+</dict>
+</plist>
```
Comment (grese) [inline App1.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: I don’t think we need this file in the PR, do we?  You can probably just remove it from the PR.

File: App1/Analytics.storyboard [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="13142" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+    <dependencies>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="12042"/>
+    </dependencies>
+    <scenes/>
+</document>
```
Comment (grese) [inline App1/Analytics.storyboard]: We don’t need the Analytics.storyboard either, do we?  It looks to me like you added the analytics VC to the Main.storyboard, so this one shouldn’t be necessary.

**Comment by grese** [comment]
@nj-app - just a few things requested.  Looks like there are a couple of files that can be removed from the PR.

Let me know if I’m missing something.

File: App1.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (hunk 1 above)
Comment (nj-app) [reply App1.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Removed file

File: App1/Analytics.storyboard (hunk 2 above)
Comment (nj-app) [reply App1/Analytics.storyboard]: Yep, that's correct. Removed this file.

**Pull request merged**